story-generator/src/app/proving-grounds/proving-grounds.html

```
@@ -0,0 +1,464 @@
+              <option [ngValue]="option">{{ option }} chapter{{ option === 1 ? '' : 's' }}</option>
+            }
+          </select>
+        </div>
+
+        <!-- Themes -->
+        <div class="form-group">
+          <label>Themes (Select 1-3)</label>
+          <div class="theme-grid">
+            @for (theme of themeOptions; track theme.id) {
+              <button
+                class="theme-tag"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)